bcgov/moh-prime
# Fix/downgrade sentry to be compatible with oc sentry

Downgraded Sentry to version 1.2 according to Sentry Self Hosted v 9.1.2 docs, this is a compatible version.

### To test:

1. Create a local environment variable called Sentry__Dsn, use the dsn shared on slack.
2. Let me know so I can add you to the project to see the dashboard and the exceptions
3. You need to throw an exception and make sure it shows on the Sentry dashboard

# OpenShift-based method of testing

1.  Add environment variable (i.e. `Sentry__Dsn`) to ReplicationController, delete web-api pod so environment variable takes effect in new pod 
2.  Request https://pr-1940.pharmanetenrolment.gov.bc.ca/api/v1/Lookups to make sure new web-api pod is working 
3.  Rename a lookup table, e.g. ProvinceLookup
4.  Request again, should get HTTP Status 500 error, exception should be reported to Sentry dashboard, web-api pod should continue to run
